### ISSUE TITLE
[BUG] You are loading @emotion/react when it is already loaded. Running multiple instances may cause problems.

### ISSUE THREAD

**Issue by fransiscushermanto** [csandman/chakra-react-select#73]
**Describe the bug**
It's happened because of react-select have it's own react-select version

**Link to Reproduction**
[A link to a CodeSandbox with a minimal reproduction of the bug you're experiencing.](https://github.com/emotion-js/emotion/issues/2343#issuecomment-817257559)

**Screenshots**
![image](https://user-images.githubusercontent.com/42550309/158359191-0450d253-7fc2-4e49-bdf2-d6b3aca61a7e.png)

**Comment by fransiscushermanto** [comment]
@csandman I think you can try to use  this solutions https://github.com/emotion-js/emotion/issues/2343#issuecomment-900209935. I can't do anything because react-select packages is installed in your package

**Comment by csandman** [comment]
Can you share a list of your dependencies (deps, peer deps, dev deps) so I can look into it?

**Comment by csandman** [comment]
Also, is this actually causing any problems, or are you just getting the warning?

**Comment by csandman** [comment]
So I implemented a way to avoid this warning in #115.  The main idea is just that your project's `@emotion/react` version needs to be equal or higher than the version required by React Select (in this case `^11.8.1`) which will allow NPM to de-dupe the package, preventing this warning/error.

To enforce this, I set a peer dependency requirement of `@emotion/react` to be the same as what's required by React Select.
